### ISSUE TITLE
Website: functions "combined" in the reference

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite-R#1182]
https://www.open-systems-pharmacology.org/OSPSuite-R/reference/index.html

![grafik](https://user-images.githubusercontent.com/25061876/213495468-f943b053-da92-4d53-b298-22ea58f11334.png)

![grafik](https://user-images.githubusercontent.com/25061876/213495755-e2df4012-3a96-4829-87bb-5b6c4d2d5724.png)

![grafik](https://user-images.githubusercontent.com/25061876/213495964-7547f13f-768e-40fb-8152-04784287a854.png)

![grafik](https://user-images.githubusercontent.com/25061876/213496121-7af29747-d523-4bdd-be5e-a06e83ddcdc1.png)

![grafik](https://user-images.githubusercontent.com/25061876/213496248-037f2593-8492-4364-a924-07b94f094c43.png)

**Comment by IndrajeetPatil** [comment]
This was by design. 

Anyone who is using the data frame method should also be aware that there is a tibble method available, and vice versa. 
We are just flocking birds of a feather together. 🙃

**Comment by Yuri05** [comment]
Makes sense. We should extend then the descriptions to `...data.frame or tibble...`
The one remaining issue with the formula should be separated.
